main/externalData/balances/index.ts

```diff
@@ -10,6 +10,7 @@ export default function (store: Store) {
   const storeApi = {
     getActiveAddress: () => (store('selected.current') || '') as Address,
     getNetwork: (id: number) => (store('main.networks.ethereum', id) || {}) as Network,
+    getNativeCurrencySymbol:(id: number) => store('main.networksMeta.ethereum', id, 'nativeCurrency', 'symbol') as string || '?',
```

Comment:
I think this is a change in functionality, we only need to set the symbol to `?` for display purposes in the UI (also I don't think it should ever really be missing but that ? code might just be older checks)